frontend/src/app/settings/modelsources/t4c-settings/edit-t4c-instance/edit-t4c-instance.component.html

```diff
@@ -140,8 +140,11 @@ <h2 *ngIf="!existing">Add a Team4Capella instance</h2>
     </mat-card>
   </div>
 
-  <app-t4c-instance-settings
-    *ngIf="this.instance"
-    [instance]="instance"
-  ></app-t4c-instance-settings>
+  <div>
+    <app-licences *ngIf="this.instance" [instance]="instance"></app-licences>
```

Comment:
`app-licenses` is still in the `session-overview` folder. Please refactor it and move it to the new location.